src/Fixer/ConfigurationDefinitionFixerInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer;
+
+use PhpCsFixer\OptionsResolver;
+
+/**
+ * @deprecated Will be merged into ConfigurableFixerInterface in 3.0
+ */
+interface ConfigurationDefinitionFixerInterface extends ConfigurableFixerInterface
```

Comment:
I'm not sure is it a good way of doing things...

We have:
- `ConfigurableFixerInterface`
- `ConfigurationDefinitionFixerInterface extends ConfigurableFixerInterface`

what we want is: later is newest form how we want to specif configuration, all should stop using original one
what we does: newly added interface is deprecated, so one should not use it. as one won't use it, he will still use original interface


We should not add interface, that is deprecated from the very beginning of its existence.
We could add new interface and deprecate existing one.